### PR TITLE
point to the good kube config in scenario 3

### PR DESCRIPTION
**Pull request by aureamunoz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ Next, you can create workspaces and start to setup logical clusters that kcp wil
 - kind is installed (if you plan to use a kind k8s cluster)
 - A k8s cluster is up and running (e.g `kind create cluster`, ...)
 - [kubectx](https://github.com/ahmetb/kubectx)
+- helm installed
```
Comment (cmoulliard) [inline README.md]: Can you add also for kind and helm the links to the their web project ?

**Review by cmoulliard — COMMENTED** [review]
LGTM - see my question

File: README.md (hunk 1 above)
Comment (aureamunoz) [reply README.md]: done

**Comment by aureamunoz** [comment]
We can merge this one, right?

**Pull request merged**